### ISSUE TITLE
Eval data NOT FOUND

### ISSUE THREAD

**Issue by bagustris** [nttcslab/dcase2023_task2_baseline_ae#14]
I tried to download eval data via `data_download_2024eval.sh` but it failed due to `404 NOT FOUND`.

```
dcase2024_task2(main)$ ./data_download_2024eval.sh            [24/9312]
--2024-06-03 09:24:13--  https://zenodo.org/records/11363076/files/eval_data_ban
dsaw_test.zip
Resolving zenodo.org (zenodo.org)... 188.185.79.172, 188.184.103.159, 188.184.98
.238, ...
Connecting to zenodo.org (zenodo.org)|188.185.79.172|:443... connected.
HTTP request sent, awaiting response... 404 NOT FOUND
2024-06-03 09:24:15 ERROR 404: NOT FOUND.

unzip:  cannot find or open eval_data_bandsaw_test.zip, eval_data_bandsaw_test.z
ip.zip or eval_data_bandsaw_test.zip.ZIP.
--2024-06-03 09:24:15--  https://zenodo.org/records/11363076/files/eval_data_gri
nder_test.zip
Resolving zenodo.org (zenodo.org)... 188.184.98.238, 188.184.103.159, 188.185.79
.172, ...
Connecting to zenodo.org (zenodo.org)|188.184.98.238|:443... connected.
HTTP request sent, awaiting response... 404 NOT FOUND
2024-06-03 09:24:16 ERROR 404: NOT FOUND.

unzip:  cannot find or open eval_data_grinder_test.zip, eval_data_grinder_test.z
ip.zip or eval_data_grinder_test.zip.ZIP.
--2024-06-03 09:24:16--  https://zenodo.org/records/11363076/files/eval_data_sha
ker_test.zip
Resolving zenodo.org (zenodo.org)... 188.184.98.238, 188.184.103.159, 188.185.79
.172, ...
Connecting to zenodo.org (zenodo.org)|188.184.98.238|:443... connected.
HTTP request sent, awaiting response... 404 NOT FOUND
2024-06-03 09:24:17 ERROR 404: NOT FOUND.

unzip:  cannot find or open eval_data_shaker_test.zip, eval_data_shaker_test.zip
.zip or eval_data_shaker_test.zip.ZIP.
--2024-06-03 09:24:17--  https://zenodo.org/records/11363076/files/eval_data_Toy
Drone_test.zip
Resolving zenodo.org (zenodo.org)... 188.184.98.238, 188.184.103.159, 188.185.79
.172, ...
Connecting to zenodo.org (zenodo.org)|188.184.98.238|:443... connected.
HTTP request sent, awaiting response... 404 NOT FOUND
2024-06-03 09:24:19 ERROR 404: NOT FOUND.

unzip:  cannot find or open eval_data_ToyDrone_test.zip, eval_data_ToyDrone_test
.zip.zip or eval_data_ToyDrone_test.zip.ZIP.
--2024-06-03 09:24:19--  https://zenodo.org/records/11363076/files/eval_data_Toy
Nscale_test.zip
Resolving zenodo.org (zenodo.org)... 188.184.98.238, 188.184.103.159, 188.185.79
.172, ...
Connecting to zenodo.org (zenodo.org)|188.184.98.238|:443... connected.
HTTP request sent, awaiting response... 404 NOT FOUND
2024-06-03 09:24:20 ERROR 404: NOT FOUND.
...
```

It seems the error is caused by different file names between the bash file and Zenodo.
@noboru2000 @YuriMusashijima

**Comment by noboru2000** [comment]
@bagustris 
Thank you for your report and sorry for the inconvenience.
We have updated the script with the latest release (v3.2.1).
Check it out!

**Comment by noboru2000** [comment]
Fixed with v3.2.1.
